extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/configproperties/ConfigPropertyBuildItemCandidate.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.arc.deployment.configproperties;
+
+public class ConfigPropertyBuildItemCandidate {
```

Comment:
Minor nitpick:

I assume that the fields of this class can be made final and we can do away with the setters, correct?

Reply:
You are absolutely right! Fixed.

Reply:
Cool, thanks